sql/7_0_2-to-7_0_3_upgrade.sql

```diff
@@ -327,10 +327,3 @@ UPDATE list_options SET title='Subcutaneous' WHERE list_id='drug_route' AND titl
 #IfRow2D list_options list_id drug_interval title q.d.
 UPDATE list_options SET title='Daily' WHERE list_id='drug_interval' AND title='q.d.';
 #EndIf
-
-#IfNotIndex sct2_description idx_term
-ALTER TABLE sct2_description
-    ADD INDEX idx_term (term),
-    ADD INDEX idx_active_term (active, term),
-    ADD FULLTEXT INDEX ft_term (term);
-#EndIf
```

Comment:
Since this is going to be in initial 7.0.3 release, it should stay in this file :)

Reply:
pretty sure version.php $v_database should be incremented, though, in master and rel-703

Reply:
but initial `7.0.3` release will be built off of `rel-703` branch

Reply:
hi. The rel-703 branch does not have the sql/7_0_3-to-7_0_4_upgrade.sql script, which is why it goes into sql/7_0_2-to-7_0_3_upgrade.sql script in master and rel-703 (these upgrade sql scripts do not ever differ across different versions of OpenEMR).
Now when we get into patches, then it is different. For a patch, it would go to sql/7_0_3-to-7_0_4_upgrade.sql in master and then in rel-703 would go into patch.sql.
Hopefully that makes sense. Late here and my brain slow :)

Reply:
makes sense and think the only minor issue is for people who checked out rel-703 before the release will have to go back and re-run the upgrade from 7.02

Reply:
whoops, not before the release but before pr 8040 was brought in